src/graphql/schema/buildObjectType.ts

```diff
@@ -436,11 +437,11 @@ function buildObjectType(payload: Payload, name: string, fields: Field[], parent
     array: (objectTypeConfig: ObjectTypeConfig, field: ArrayField) => {
       const fullName = combineParentName(parentName, field.label === false ? toWords(field.name, true) : field.label);
       const type = buildObjectType(payload, fullName, field.fields, fullName);
-      const arrayType = new GraphQLList(withNullableType(field, type));
```

Comment:
this withNullableType call seemed to be misplaced, it was making each element non nil instead of the array type